internal/routes.go

```diff
@@ -140,6 +142,16 @@ func fetchMovie(id string, movieCh chan Movie) {
 
 	defer res.Body.Close()
 
+	if res.StatusCode == 429 {
```

Comment:
can we make use of the http built-in constants? e.g. `http.StatusOK`

Reply:
Ah, good catch! I'm changing 429 to `http.StatusTooManyRequests` as well

Reply:
Awesome 🙌